src/Components/Radio/Radio.component.jsx

```diff
@@ -77,14 +78,29 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const [status, setStatus] = useState(0);
   const [audio, setAudio] = useState(null);
   const [playing, setPlaying] = useState(false);
-  let nowPlaying = '<Song Name>';
-
-  if (activeTrack) {
-    nowPlaying = activeTrack.title;
-  }
+  //let nowPlaying = '<Song Name>';
+  const [nowPlaying, setNowPlaying] = useState('<CLICK PLAY!>');
+
+  const siteUrl = 'https://icecast.softboys.club:18000';
+  const fetchData = async () => {
+    //NEED ICECAST TRACK INDEX
+    if(activeTrack) {
+      const icecastIndex = activeTrack.id.slice(-1);
+
+      const result = await axios.get(
+        siteUrl + "/status-json.xsl"
+      );
+
+      //IF ONE SOURCE, ICESTATS HAS NO ARRAY
+      if(Array.isArray(result.data.icestats.source))
```

Comment:
nit

- lets keep this control syntax in line with the rest of the app and bracket it

Reply:
the if, else?